test/e2e/specs/site-editor/style-book.spec.js

```diff
@@ -119,7 +119,7 @@ test.describe( 'Style Book', () => {
 
 	test( 'should disappear when closed', async ( { page } ) => {
 		await page.click(
-			'role=region[name="Style Book"i] >> role=button[name="Close Style Book"i]'
+			'role=region[name="Style Book"i] >> role=button[name="Style Book"i]'
```

Comment:
The button is the "X" close icon in the style book, not in the global styles sidebar. We don't need to change this :).

```suggestion
			'role=region[name="Style Book"i] >> role=button[name="Close Style Book"i]'
```

Reply:
Ah, whoops, thanks @kevin940726! 🙂